src/thumbnail-image/index.md

```diff
@@ -0,0 +1,31 @@
+# What is a thumbnail?
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096). 
+- The aspect ratio must be 16x9. If it's not 16x9, you can use the handy cropping tool that we've provided. Alternatively, you can crop it with your tool of choice before uploading. 
+    - _Tip:_ This will help if you're not using our cropping tool. You can find the necessary height by taking the image's current width and multiplying it by 0.5625. Round up. 
```

Comment:
They don't have to use the cropping tool if their image is the right size. Is that correct, @hworld ?